### DISCUSSION TITLE
How can i run steam on Box64

### DISCUSSION THREAD

**Discussion by Toddo0** [ptitSeb/box64#524] [Q&A]
This guy claims that box64 runs steam. i tried but it dosnt work. i didnt undurstant what he is really do but seems like its working. i am sure that in github page says no steam for box64. but if its possiable how can i do?

video link: https://www.youtube.com/watch?v=co3Qi9ZwS5Q&t=16s

Note: i tried on Snapdiragon 750G(SM7225) using termux 
OS->debian 11(xfce)

**Comment by ptitSeb** [comment]
You need both box86 & box64 to run steam. And you need to build the `steam_chrome`  branch of box64 also.

**Comment by Toddo0** [comment]
İ have some issues 

`
```
root@localhost:~/box64/build# cmake .. -DARM_DYNAREC=ON -DCMAKE_BUILD_TYPE=RelWithDebInfo; make -j$(nproc)
-- Configuring done
-- Generating done
-- Build files have been written to: /root/box64/build
[  2%] Built target dynarec_native
[  2%] Built target WRAPPERS
[  9%] Built target native_pass0
[ 16%] Built target native_pass3
[ 23%] Built target native_pass1
[ 30%] Built target native_pass2
[ 31%] Built target dynarec
[ 31%] Linking C executable box64
/usr/bin/ld: libdynarec.a(dynarec_native_pass.c.o): in function `native_pass3':
/root/box64/src/dynarec/dynarec_native_pass.c:162: undefined reference to `dynarec64_003'
collect2: error: ld returned 1 exit status
make[2]: *** [CMakeFiles/box64.dir/build.make:4157: box64] Error 1
make[1]: *** [CMakeFiles/Makefile2:332: CMakeFiles/box64.dir/all] Error 2
make: *** [Makefile:163: all] Error 2
```
`

Also ctest fails on everything

**Reply by ptitSeb** [reply]
`make clean && make` from the build folder to rebuild from scratch. Something went wrong (also, check you have enough disk space)
